security/providers/oidc-common/src/main/java/io/helidon/security/providers/oidc/common/OidcConfig.java

```diff
@@ -1184,7 +1224,8 @@ public Builder forceHttpsRedirects(boolean forceHttpsRedirects) {
          * @param relativeUris relative URIs flag
          * @return updated builder instance
          */
-        @ConfiguredOption("false")
+        @ConfiguredOption(value = "false", deprecated = true)
+        @Deprecated(since = "4.2.1", forRemoval = true)
```

Comment:
Whenever you deprecate something (esp. when `forRemoval=true`), you need to add a javadoc `@deprecated` that explains what to do instead

Reply:
Please fix all in thisi class.